test/com/google/enterprise/adaptor/sharepoint/SharePointAdaptorTest.java

```diff
@@ -150,12 +146,14 @@
   private static final String VS_ENDPOINT
       = "http://localhost:1/_vti_bin/SiteData.asmx";
   private static final ContentExchange VS_CONTENT_EXCHANGE
-      = new ContentExchange(ObjectType.VIRTUAL_SERVER, null, null, null,
-          true, false, null, loadTestString("vs.xml"));
+      = new ContentExchange.Builder().setObjectType(ObjectType.VIRTUAL_SERVER)
+          .setRetrieveChildItems(true).setSecurityOnly(false)
```

Comment:
I'm not sure about multiple setters on one line, since they're all kind of long. I think I'd try new ContentExchange.Builder()\n...\n.build(); as a pattern, too.

Reply:
Done